test/test_data/rrule_until_only_date.ics

```diff
@@ -0,0 +1,43 @@
+BEGIN:VCALENDAR
+PRODID:-//Google Inc//Google Calendar 70.9054//EN
+VERSION:2.0
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+X-WR-CALNAME:brian.hartvigsen@charthop.com
```

Comment:
hope you are ok with your email in this test :-)

Reply:
@tresni let me know if we can keep this email in here. If yes, I will merge this pr.

Reply:
damn it... totally thought I'd sanitized everything... I'll fix that if you don't mind ;-)  Won't clean it out of the git history but...